view/src/components/common/EventParticpantCard/EventParticpantCard.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import Paper from '@mui/material/Paper';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TablePagination from '@mui/material/TablePagination';
+import TableRow from '@mui/material/TableRow';
+import Box from '@mui/material/Box'
+import { useState, useEffect } from 'react';
+
+interface User {
+  id: number
+  name: string
+  number: number
+  created_at: number
+  updated_at: number
+};
+
+interface Events {
+  id: number
+  name: string
+  description: string
+  max_attendee: number
+  start_at: number
+  end_at: number
+  created_at: number
+  updated_at: number
+  user: User[]
+};
+
+const EventParticpantCard = () => {
+  const [eventusers, setEventUsers] = useState<Events[]>()
+
+  useEffect(() => {
+    fetch(process.env.CSR_API_URI + '/events/7/users', {
```

Comment:
ここは直接値を入れるのではなく、propsから取って来たいです
propsはcomponentの引数みたいなもので、そのcomponentを使ってる親から値を受け取ることができます！
手順を示します。

1.` common/EventParticipantCard/`内に`EventParticipantCard.type.ts`を作成
2. propsの型定義をそのファイルに書く

```
export interface EventParticipantCardProps {
    eventId: string;
}
```

これぐらいだったら直接`EventParticipantCard.tsx`に書けばいいじゃん！と思うかもしれませんが、propsで取ってきたい変数が増えるほど別ファイルに定義したほうが綺麗になります！

また、`User`や`Event`みたいな型定義も`EventParticipantCard.type.ts`に書いたほうが綺麗になります！そうするなら、`.type.ts`じゃなく`.types.ts`にしたほうが意味が近いのでそうしましょう！

3. `EventParticipantCard.tsx`でimport

```
import { EventParticipantCardProps } from './EventParticipantCard.type';
```

4. EventParticipantCardにpropsを与える

```
const EventParticipantCard = (props: EventParticipantCardProps) => {
```

5. fetchしたい行でpropsを使う

```suggestion
    fetch(process.env.CSR_API_URI + `/events/${props.eventId}/users`, {
```

バッククォートの中に`${}`で変数を囲むと文字列の中に組み込むことができます！